README.md

```diff
@@ -89,6 +89,7 @@ Read some additional details about 'moderator mode' on the [wiki page here](http
 ## Installation
 
 If using the python script version (not the exe), there is a requirements.txt with necessary modules. Created with Python 3.9.7
+If your using the exe version (not the python script) on a Windows computer, under Windows 8, you will not be able to run the exe version because dll (a component of this project) was only added in Windows 8 and above. Although, the Python Script should work fine, incase of any issues, feel free to report bugs [here](https://github.com/ThioJoe/YT-Spammer-Purge/issues/new/choose) 
```

Comment:
```suggestion
Windows exe version requires Windows 8 or above [due of the packaging app used](https://pyinstaller.readthedocs.io/en/stable/requirements.html#windows)
```

Reply:
@EthanHindmarsh I dont think @ThioJoe can add this requirement in all releases, users are required to read README so yeah, if they dont and they wonder what the error is, its their fault xD

Reply:
On second thought,  I dont even think Why is required, but let ThioJoe decide on that, leaving it in for now.